### PR TITLE
add clear on touchstart

### PR DESCRIPTION
**Pull request by davidpett**
currently on iOS, the clear button opens the dropdown instead of clearing the selection. this adds the touch support

### CODE REVIEW

**Comment by sebastianseilund** [comment]
This patch makes the clear button work on iOS in my app as well.

@davidpett could you make the tests pass? I think if you add `this.foo = 'one';` after `this.numbers = numbers;` then it'll work. The problem right now is that the clear button doesn't appear since nothing has been selected.

Then I think @cibernox can merge and we can get this in the next release.

Thanks!

**Comment by cibernox** [comment]
There is a ESLint error, could you rebase and fix it?

File: addon/components/power-select/trigger.js [hunk 1]
```diff
@@ -10,6 +10,9 @@ export default Component.extend({
     clear(e) {
       e.stopPropagation();
       this.get('select').actions.select(null);
+      if (e.type === 'touchstart') {
```
Comment (cibernox) [inline addon/components/power-select/trigger.js]: Why do you return false here?

Comment (davidpett) [inline addon/components/power-select/trigger.js]: to not bubble the event to the click which would open the dropdown

**Comment by cibernox** [comment]
Thanks! I'll release a new version soon

**Pull request merged**